lib/web_ui/test/canvaskit/surface_factory_test.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:test/bootstrap/browser.dart';
+import 'package:test/test.dart';
+import 'package:ui/src/engine.dart';
+
+import '../matchers.dart';
+import 'common.dart';
+
+const MethodCodec codec = StandardMethodCodec();
+
+void main() {
+  internalBootstrapBrowserTest(() => testMain);
+}
+
+void testMain() {
+  group('$SurfaceFactory', () {
```

Comment:
Let's also add tests for `getSurface`, `releaseSurface`, and `isLive`. Or if they are covered by another test, let's leave a comment pointing to that test, although I'd prefer that the class is tested standalone, if possible.

Reply:
Done